packages/ui/components/badge/UpgradeOrgsBadge.tsx

```diff
@@ -0,0 +1,16 @@
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+
+import { Tooltip } from "../tooltip";
+import { Badge } from "./Badge";
+
+export const UpgradeOrgsBadge = function UpgradeOrgsBadge() {
+  const { t } = useLocale();
+
+  return (
+    <Tooltip content={t("orgs_upgrade_to_enable_feature")}>
+      <a href="https://cal.com/enterprise" target="_blank">
```

Comment:
are we ok with linking to https://cal.com/enterprise when clicking on the upgrade badge? 
<img width="916" alt="Screenshot 2023-09-15 at 11 43 23" src="https://github.com/calcom/cal.com/assets/30310907/8605942f-ceff-4351-b09f-a0abb4f5149c">